scripts/measures/2023/update-measures.ts

```diff
@@ -10,223 +10,138 @@
 
 import _ from 'lodash';
 import fs from 'fs';
-import parse from 'csv-parse/lib/sync';
 import path from 'path';
 import appRoot from 'app-root-path';
 
-import { info, error, warning} from '../../logger';
+import { info, error, warning } from '../../logger';
 import { initValidation, MeasuresChange, measureType } from '../lib/validate-change-requests';
-import * as Constants from '../../constants';
+import { convertCsvToJson } from '../lib/csv-json-converter';
+import { DataValidationError } from '../lib/errors';
 
 const performanceYear = process.argv[2];
 
 const measuresPath = `measures/${performanceYear}/measures-data.json`;
 const changesPath = `updates/measures/${performanceYear}/`;
 
 const measuresJson: any[] = JSON.parse(
-    fs.readFileSync(path.join(appRoot+'', measuresPath), 'utf8')
+    fs.readFileSync(path.join(appRoot + '', measuresPath), 'utf8')
 );
 
 const changelog = JSON.parse(
-    fs.readFileSync(path.join(appRoot+'', `${changesPath}Changelog.json`), 'utf8')
+    fs.readFileSync(path.join(appRoot + '', `${changesPath}changes.meta.json`), 'utf8')
 );
 
-//to determine if any new changes need to be written to measures-data.json.
-let numOfNewChangeFiles = 0;
 
 function updateMeasures() {
-    const files = fs.readdirSync(path.join(appRoot+'', changesPath));
+    //to determine if any new changes need to be written to measures-data.json.
+    let numOfNewChangeFiles = 0;
 
-    files.forEach(fileName => {
-        //find only the change files not yet present in the changelog.
-        if(fileName != 'Changelog.json') {
-            if(!changelog.includes(fileName)) {
+    const fileNames = fs.readdirSync(path.join(appRoot + '', changesPath));
+
+    fileNames.forEach(fileName => {
+        if (fileName != 'changes.meta.json') {
+            //find only the change files not yet present in the changelog.
+            if (!changelog.includes(fileName)) {
                 numOfNewChangeFiles++;
                 updateMeasuresWithChangeFile(fileName)
             }
         }
     });
 
-    if(numOfNewChangeFiles > 0) {
+    if (numOfNewChangeFiles > 0) {
         writeToFile(measuresJson, measuresPath);
     } else {
         info(`No new change files found.`);
     }
 }
 
-//not needed once we only accept JSON change requests.
```

Comment:
The ConvertCsvToJson logic was moved to it's own script, along with its helper functions.